README.md

```diff
@@ -2,3 +2,62 @@
 
 Support drupal module for [uk.co.compucorp.membershipextras](https://github.com/compucorp/uk.co.compucorp.membershipextras) extension that handle the creation of upfront contributions after submitting a civicrm webform.
 
+## Do I need This Module?
+If you're using membership extras extension to manage memberships and payment plans, and you would like to allow your users to sign-up for those memberships via a webform, then yes! This module is required, as it will add the necessary form controls to let visitors to your site to choose the number of installments in which they would like to pay for the membership and handle the creation of those contributions and associate them to the payment plan.
+
+## Requirements
+This module works in tandem with membership extras CiviCRM extension. Aside from that, you will also need to install a patched version of the [CiviCRM webforms module v7.x-4.28](https://github.com/compucorp/webform_civicrm/releases/tag/7.x-4.28-patch1) which has several features and fixes done in order to support the creation of payment plans and memberships via a webform:
+
+- [Patch #1](https://github.com/colemanw/webform_civicrm/pull/180): Use Selected Payment Processor, Instead Default
+- [Patch #2](https://github.com/colemanw/webform_civicrm/pull/274): Support for Membership Autorenewal
+- [Patch #3](https://github.com/colemanw/webform_civicrm/pull/275): Improve Autorenewal UX
+- [Patch #4](https://github.com/colemanw/webform_civicrm/pull/280): Improve Accuracy Calculations Done on Values with Decimals
+- [Patch #5](https://github.com/colemanw/webform_civicrm/pull/282): Fixes Line Items View When Selecting Memberships via Checkbox
+- [Patch #6](https://github.com/colemanw/webform_civicrm/pull/284): Autofills Related Contacts on Same Page
+
+### Patches
```

Comment:
These patches are related to our fork of webform_civicrm module, and not this module. So I suggest to move them there, you can create a PR against this branch : https://github.com/compucorp/webform_civicrm/tree/7.x-4.28-patches

Reply:
The instruction by Jamie was to document the patches made on CiviCRM core on the extension, and the patches made on webform_civicrm module on the webform_civicrm_membership_extras module, which I think makes sense, as it explains to someone wanting to install the ME module why it needs to be installed on a patched version of the civicrm webform module.

Reply:
I think Jamie's comment was more of an assumption rather than an instruction, anyway I see your point.. I think it is fine for now then.